view/metadata.go

```diff
@@ -58,6 +58,39 @@ func LoadMetadata(ctx context.Context,
 	return &m, nil
 }
 
+// NewMetadata returns a view metadata for a given version.
+func NewMetadata(schema *iceberg.Schema,
+	version Version,
+	loc string,
+	props iceberg.Properties,
+) Metadata {
+	// override the corresponding view version fields with the following values.
+	versionId := int64(1)
```

Comment:
why are we overriding the version.VersionID with 1?

Reply:
Can remove, was thinking it may be better to have a predictable starting point vs whatever is passed in